### ISSUE TITLE
[feat] Implement Ctrl-R to open query history

### ISSUE THREAD

**Issue by hshimizu** [axonops/axonops-workbench#217]
![image](https://github.com/user-attachments/assets/92f33064-2556-4e42-8c3e-be32c3f2e73e)

When I have selected the field to enter CQL statement, I'd like the UNIX Ctrl-R combination to show/search the history.

Up/Down key should also work to go through the history.

**Comment by millerjp** [comment]
@hshimizu I dont think this is possible - theres an issue with that Ctrl-R. I cant quite rememver what it was @mhmdkrmabd ?

**Comment by mhmdkrmabd** [comment]
@millerjp @hshimizu  I think you're mentioning macOS as it reserves the CTRL (UP/DOWN) arrow keys, for CTRL-R I believe there's no issues, I like this option on Linux as well, maybe we can add one in the following way:
- User presses CTRL-R combination.
- The history list will show up (we'll add a search input field at the bottom), it'll be auto focused.
- As the user types in this input field the related statements will be added to the cql statements input.

**Comment by millerjp** [comment]
@mhmdkrmabd that sounds like a good approach

**Comment by millerjp** [comment]
Closing as histroy and UX has beem updated since issue raised.
